pyglet/lib.py

```diff
@@ -164,6 +164,9 @@ def __init__(self):
         else:
             self.dyld_fallback_library_path = [os.path.expanduser('~/lib'), '/usr/local/lib', '/usr/lib']
 
+        # check homebrew for libs too
+        self.brew_library_path = ['/opt/homebrew/lib']
```

Comment:
This might be better as a keyword argument. The doc seems to strongly recommend against changing the directory, [but it does seem to support it in theory](https://docs.brew.sh/Installation#untar-anywhere-unsupported) if I understood it correctly.